lib/datadog/appsec/processor.rb

```diff
@@ -16,9 +16,38 @@ def self.===(other)
         end
       end
 
+      # Context wraps libddwaf's context
+      class Context
```

Comment:
Minor: In the spirit of self-documenting code, have you considered perhaps naming this `ContextWrapper` and removing the comment? ;)

Reply:
I bet removing the comment will make Rubocop cry :/, but the renaming is a good suggestion.

Reply:
I guess it would be *exact* but I don't feel it would be *useful*: it would only expose internals from an abstraction that is supposed to hide what is happening behind the scenes and augment it with additional features.

This makes me think it's in fact the comment that is incorrect now.